pkg/webhooks/validation/server/server_test.go

```diff
@@ -223,6 +251,57 @@ func TestAdmitPilot(t *testing.T) {
 			}
 		})
 	}
+
+	smeV1 := makeServiceMeshExtensionConfig(t, "v1")
+	smeV1alpha1 := makeServiceMeshExtensionConfig(t, "v1alpha1")
+	deprecatedWarning := "ServiceMeshExtension is deprecated. Please use WasmPlugin instead."
+
+	maistraCases := []struct {
+		name string
+		in   *kube.AdmissionRequest
+	}{
+		{
+			name: "Create v1 ServiceMeshExtension",
+			in: &kube.AdmissionRequest{
+				Kind:      kubeApisMeta.GroupVersionKind{Kind: "ServiceMeshExtension"},
+				Object:    runtime.RawExtension{Raw: smeV1},
+				Operation: kube.Create,
+			},
+		},
+		{
+			name: "Update v1 ServiceMeshExtension",
+			in: &kube.AdmissionRequest{
+				Kind:      kubeApisMeta.GroupVersionKind{Kind: "ServiceMeshExtension"},
+				Object:    runtime.RawExtension{Raw: smeV1},
+				Operation: kube.Update,
+			},
+		},
+		{
+			name: "Create v1alpha1 ServiceMeshExtension",
+			in: &kube.AdmissionRequest{
+				Kind:      kubeApisMeta.GroupVersionKind{Kind: "ServiceMeshExtension"},
+				Object:    runtime.RawExtension{Raw: smeV1alpha1},
+				Operation: kube.Create,
+			},
+		},
+		{
+			name: "Update v1alpha1 ServiceMeshExtension",
+			in: &kube.AdmissionRequest{
+				Kind:      kubeApisMeta.GroupVersionKind{Kind: "ServiceMeshExtension"},
+				Object:    runtime.RawExtension{Raw: smeV1alpha1},
+				Operation: kube.Create,
```

Comment:
Operation should be `kube.Update` in this case.

Reply:
Done